test/exampleGeneratorTests.ts

```diff
@@ -107,7 +107,14 @@ describe("test generate example", () => {
 });
 
 export function getSpecFilePaths(repoDir: string) {
-  const rpList = fs.readdirSync(path.resolve(repoDir, "specification"));
+  let rpList: string[];
```

Comment:
jest@27 included change to run the delegate inside a "skipped" suite, and just skip the test cases.  This caused our skipped test to fail in jest@27, until this fix.